eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -52,8 +53,25 @@ relevantDuringBootstrapSpec ::
   forall era.
   ConwayEraImp era =>
   SpecWith (ImpTestState era)
-relevantDuringBootstrapSpec =
+relevantDuringBootstrapSpec = do
   spoVotesForHardForkInitiation
+  initiateHardForkWithLessThanMinimalCommitteeSize
+
+initiateHardForkWithLessThanMinimalCommitteeSize ::
+  forall era.
+  ConwayEraImp era =>
+  SpecWith (ImpTestState era)
+initiateHardForkWithLessThanMinimalCommitteeSize =
+  it "Hard Fork can still be initiated with less than minimal committee size" $ do
+    (hotC :| _) <- registerInitialCommittee
+    passEpoch
+    modifyPParams $ ppCommitteeMinSizeL .~ 2
+    protVer <- getProtVer
+    gai <- submitGovAction $ HardForkInitiation SNothing (majorFollow protVer)
+    submitYesVote_ (CommitteeVoter hotC) gai
+    if bootstrapPhase protVer
+      then isCommitteeAccepted gai `shouldReturn` True
```

Comment:
I'd recommend adding SPO votes and also checking that that proposal gets enacted. This will require registering and voting with DReps outside of bootstrap phase.

But, that is not strictly a requirement. It would just make this test a bit more robust and complete

Reply:
Sounds good! I'll do this in a separate PR as well.